index.html

```diff
@@ -85,24 +75,12 @@
 			<article id="ts-root-toggle">
 				<h2>ts-root</h2>
 				<ts-button-group>
-					<ts-button onclick="toggleSlot('header', this)"
-						>Toggle Header</ts-button
-					>
-					<ts-button onclick="toggleSlot('footer', this)"
-						>Toggle Footer</ts-button
-					>
-					<ts-button onclick="toggleSlot('sidebar-left', this)"
-						>Toggle Left Sidebar</ts-button
-					>
-					<ts-button onclick="toggleSlot('sidebar-right', this)"
-						>Toggle Right Sidebar</ts-button
-					>
-					<ts-button onclick="toggleSlot('sidebar-inner-left', this)"
-						>Toggle Inner Left Sidebar</ts-button
-					>
-					<ts-button onclick="toggleSlot('sidebar-inner-right', this)"
-						>Toggle Inner Right Sidebar</ts-button
-					>
+					<ts-button onclick="toggleSlot('header', this)">Toggle Header</ts-button>
+					<ts-button onclick="toggleSlot('footer', this)">Toggle Footer</ts-button>
+					<ts-button onclick="toggleSlot('sidebar-left', this)">Toggle Left Sidebar</ts-button>
+					<ts-button onclick="toggleSlot('sidebar-right', this)">Toggle Right Sidebar</ts-button>
+					<ts-button onclick="toggleSlot('sidebar-inner-left', this)">Toggle Inner Left Sidebar</ts-button>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a 'onKeyPress' attribute to this <ts-button> tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AMouseEventWithoutKeyboardEquivalentCheck)